R/epipredict-data.R

```diff
@@ -51,31 +55,72 @@
 #'   \item{abbr}{Postal abbreviation for the location}
 #' }
 #'
-#' @source United States Census Bureau, at
+#' @source United States Census Bureau, Population Division, at
 #'   \url{https://www2.census.gov/programs-surveys/popest/datasets/2010-2019/counties/totals/co-est2019-alldata.pdf},
 #'   \url{https://www.census.gov/data/tables/time-series/demo/popest/2010s-total-puerto-rico-municipios.html},
 #'   and \url{https://www.census.gov/data/tables/2010/dec/2010-island-areas.html}
+#'   This data is public domain.
 "state_census"
 
 # Epipredict Vignette Data ----------------------------------------------------
 
-#' CTIS COVID Behaviours
+#' Subset of CTIS COVID-19 behaviours from 5 states
 #'
 #' Data set for a handful of states on masking and distancing behaviours
-#' during the COVID-19 Pandemic and downloaded from the CMU Delphi COVIDcast
-#' Epidata API. This data set covers the period from
-#' June to December 2021.
+#' during the COVID-19 Pandemic, and downloaded from the CMU Delphi COVIDcast
+#' Epidata API. This data set covers the period from June to December 2021,
+#' and is limited to California, Florida, Texas, New Jersey, and New York.
+#'
+#' @format An object of class \code{tbl_df} (inherits from \code{tbl}, \code{data.frame}) with 1055 rows and 4 columns.
+#' @section Data dictionary:
+#' A tibble with columns:
+#' \describe{
+#'   \item{geo_value}{the geographic value associated with each row
+#'       of measurements.}
+#'   \item{time_value}{the time value associated with each row of measurements.}
+#'   \item{masking}{Estimated percentage of people who wore a mask for most or all of the time while in public in the past 7 days; those not in public in the past 7 days are not counted.}
+#'   \item{distancting}{Estimated percentage of respondents who reported that all or most people they enountered in public in the past 7 days maintained a distance of at least 6 feet. Respondents who said that they have not been in public for the past 7 days are excluded.}
+#' }
+#'
+#' @source Data from Delphi COVIDcast. Obtained via the Delphi Epidata API.
+#'   \url{https://cmu-delphi.github.io/delphi-epidata/api/covidcast.html}. This
+#'   object contains a modified part of the
+#'   \href{https://cmu-delphi.github.io/delphi-epidata/symptom-survey/#covid-19-trends-and-impact-survey}{data
+#'   aggregations in the API} that are prepared from the
+#'   \href{https://www.pnas.org/doi/full/10.1073/pnas.2111454118}{COVID-19
+#'   Trends and Impact Survey}; see the first link for more information on
+#'   citing in publications. These aggregations are licensed under the terms of
+#'   the \href{https://creativecommons.org/licenses/by/4.0/}{Creative Commons
+#'   Attribution license}.
+#'
+#' Modifications:
+#' * Selected a small subset of the available data.
+#' * Changed to wide format.
+#'
+#' Delphi Group at Carnegie Mellon University
```

Comment:
issue: This `Delphi Group` line is dangling. Was it supposed to be the license-holder, the copyright-holder, something else?

Reply:
Hmm, I don't remember.  Maybe I copied it in as a potential good-sounding name before I came across the [requested attribution text for novel data](https://cmu-delphi.github.io/delphi-epidata/api/covidcast_licensing.html#novel-data).  I think we can drop it based on 3a of [CC BY 4.0](https://creativecommons.org/licenses/by/4.0/legalcode), even though I don't think the requested attribution is that clear that this is not just re-served data.